src/Microsoft.AspNetCore.Identity.Service.KeyVault/KeyVaultSigningCredentialsSourceOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net.Http;
+
+namespace Microsoft.AspNetCore.Identity.Service.KeyVault
+{
+    public class KeyVaultSigningCredentialsSourceOptions
+    {
+        public string ClientId { get; set; }
+        public string ClientSecret { get; set; }
+        public string CertificateThumbprint { get; set; }
```

Comment:
@pakrym I would keep the config simple (and bindable) as much as possible, and have the component load the certificate. This version only supports ClientSecret for now